brewster/bensconv.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
-from brewster.bbconv import prism
-from brewster.bbconv import convfwhm
-from brewster.bbconv import convr
+from .bbconv import prism
+from .bbconv import convfwhm
+from .bbconv import convr
 
```

Comment:
Will `from brewster.bbconv` still work?  Put another way, is this relative import a preference or needed because of the new build?